lib/api.js

```diff
@@ -501,9 +501,13 @@ export async function extractSavedTracks() {
 }
 
 export async function extractTrackAudioFeatures(trackIds) {
-  return await callSpotifyApi(
-    async () => (await spotifyApi.getAudioFeaturesForTracks(
-      trackIds,
-    )).body.audio_features,
-  );
+  let audioFeatures = [];
+  for(let chunk of trackIds.chunk(MAX_LIMIT_DEFAULT)) {
+    audioFeatures.push(await callSpotifyApi(
+      async () => (await spotifyApi.getAudioFeaturesForTracks(
+        chunk,
+      )).body.audio_features,
+    ));
+  }
+  return audioFeatures;
```

Comment:
```suggestion
  let audioFeatures = [];
  for(let chunk of trackIds.chunk(MAX_LIMIT_DEFAULT)) {
    audioFeatures.push(...(await callSpotifyApi(
      async () => (await spotifyApi.getAudioFeaturesForTracks(
        chunk,
      )).body.audio_features,
    )));
  }
  return audioFeatures;
```

Reply:
@pedrorault this is untested but i think we also need to unwrap the array, or concat instead of push